docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,602 @@
+https://www.ansible.com/overview/how-ansible-works[Ansible^] is an open source configuration management and automation tool.
+
+Ansible uses modules to execute tasks. The output of one module can be the input for the next one or they can be independent of each other.
+
+You can combine one or more Ansible modules to make a play. Combine two or more plays to create an Ansible Playbook, a list of sequential tasks (written in YAML) that automatically executes when you run the `ansible-playbook` command. You can use an Ansible Playbook to describe the desired state of your infrastructure and then have Ansible provision it.
+
+Ansible roles are a special kind of playbook that is fully self-contained and portable with the tasks, variables, configuration templates, and other supporting files that are needed to complete your chosen tasks. Multiple roles can exist inside a collection allowing easy sharing of content through Automation Hub and https://galaxy.ansible.com[Ansible Galaxy^].
+
+//Additional line break to resolve mod docs generation error
+
+[id="proc-install-rhoas-ansible-collection_{context}"]
+=== Installing the RHOAS Ansible collection
+
+[role="_abstract"]
+You can use the https://galaxy.ansible.com/rhoas/rhoas[RHOAS Ansible collection] to fully manage your Kafka environment with Ansible. You can create, view, configure, and delete the following {product-long-kafka} resources:
+
+* Kafka instances
+* Service accounts
+* Kafka topics
+* Access Control Lists (ACLs)
+
+.Prerequisites
+
+* https://www.python.org/downloads/[Python 3.9^] or later is installed.
+* https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html?extIdCarryOver=true&sc_cid=701f2000001Css5AAC[Ansible 2.9^] or later is installed.
+* Some additional dependencies required by this collection are installed. To install these, enter the following commands:
++
+[source,shell]
+----
+$ pip install rhoas-sdks
+$ pip install python-dotenv
+----
+NOTE: Use the `pip install rhoas-sdks --force reinstall` command to ensure that the most up to date versions of the SDKs are installed.
++
+
+* (Optional) This collection works best when used in a Python virtual environment. To create and activate a Python virtual environment, enter the following command:
++
+[source,shell]
+----
+$ python3 -m venv rhoas/
+. rhoas/bin/activate
+----
++
+
+.Procedure
+
+. Open a terminal and enter the following command:
++
+[source,shell]
+----
+$ ansible-galaxy collection install rhoas.rhoas
+----
++
+By default, the `ansible-galaxy collection install` command uses https://galaxy.ansible.com as the Galaxy server URL. The URL is specified as the value of the `GALAXY_SERVER` parameter in the `ansible.cfg` configuration file.
+. Once installed, you can reference a collection content by its fully qualified collection name (FQCN). For example the `create_kafka` command in a task needs to be the FQCN of the module for collection development, for example
+`rhoas.rhoas.create_kafka`.
```

Comment:
A couple of comments here:

- This is not really a step in the procedure - it is additional information. I think it should just be output for the preceding step.
- Before adding to the output of the preceding step, it's probably worth adding a statement to the previous step to say what happens when you run the install command. Tell users what they see if the collection is successfully installed.
- Because the procedure would become a single step, it would need to be a bullet, and not a numbered step.
- Replace "once" with "when" or "after".  See https://www.ibm.com/docs/en/ibm-style?topic=word-usage#o.
- Just wondering what's meant by "the module for collection development"?